src/test/java/org/broadinstitute/hellbender/utils/fasta/CachingIndexedFastaSequenceFileUnitTest.java

```diff
@@ -1,9 +1,7 @@
 package org.broadinstitute.hellbender.utils.fasta;
```

Comment:
Is this file actually changed in this branch? If not, remove the changes from the branch. if yes, explain the changes.